out.txt

```diff
@@ -0,0 +1,2 @@
+Hello test
```

Comment:
Again with out.txt. please read your code diff before pushing. Also let's put this in gitignore.